grobid-core/src/main/java/org/grobid/core/engines/DateParser.java

```diff
@@ -1,180 +1,214 @@
 package org.grobid.core.engines;
 
+import org.apache.commons.collections4.CollectionUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.grobid.core.GrobidModel;
 import org.grobid.core.GrobidModels;
 import org.grobid.core.data.Date;
+import org.grobid.core.engines.label.TaggingLabel;
+import org.grobid.core.engines.label.TaggingLabels;
 import org.grobid.core.exceptions.GrobidException;
 import org.grobid.core.features.FeaturesVectorDate;
-import org.grobid.core.utilities.TextUtilities;
 import org.grobid.core.lang.Language;
+import org.grobid.core.layout.LayoutToken;
+import org.grobid.core.tokenization.TaggingTokenCluster;
+import org.grobid.core.tokenization.TaggingTokenClusteror;
+import org.grobid.core.utilities.LayoutTokensUtil;
+import org.grobid.core.utilities.TextUtilities;
 
 import java.util.ArrayList;
+import java.util.Calendar;
 import java.util.List;
 import java.util.StringTokenizer;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+import static org.apache.commons.lang3.StringUtils.isNotBlank;
 
 public class DateParser extends AbstractParser {
 
     public DateParser() {
         super(GrobidModels.DATE);
     }
 
+    DateParser(GrobidModel model) {
+        super(model);
+    }
+    
     /**
-     * Processing of authors in header
-     */
+     * Deprecated, @Use process(String input)
+     **/
+    @Deprecated
     public List<Date> processing(String input) {
```

Comment:
I would keep `processing()` like everywhere else, because why just changing only this one? It's very minor difference and I think better be consistent?

Reply:
I left the `processing()` method, but deprecated it so that both can be used and the users will know that at some point will be removed. Since the change was related to the DateParser I did not modify all the parsers in this PR.

Reply:
ok I count on you then to change elsewhere then :D (I would not have the courage for all these changes just for 3 letters ;) ) 
It could be an independent PR where the method is deprecated everywhere?

Reply:
OK. Usually the IDE will do most of the job 👍